### PR TITLE
STL Export ASCII

### PR DESCRIPTION
**Pull request by Pryme8**
I reverted the .gitIgnore file changes.

I think this is correct. >_<

### CODE REVIEW

File: serializers/src/stl/stlSerializer.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import { VertexBuffer } from "babylonjs/Meshes/buffer";
+import { Vector3 } from "babylonjs/Maths/math";
+
+/**
+ * Class for generating STL data from a Babylon scene.
+ */
+export class STLExport {
+     /**
+     * Exports the geometry of a Mesh array in .STL file format (ASCII)
+     * @param mesh defines the list of meshes to serialize
```
Comment (deltakosh) [inline serializers/src/stl/stlSerializer.ts]: This is not a list right?

Comment (Pryme8) [reply serializers/src/stl/stlSerializer.ts]: good catch, fixing now.

**Comment by Pryme8** [comment]
wait whoa, so this was correct?

How would I test it on my end, I could never get it to come up on any of my builds... If I can connect this blank then I can start flying on contributions.

**Comment by deltakosh** [comment]
It did :)
the best way is to create a sample code in localdev and use vsCode to debug it

**Comment by deltakosh** [comment]
it will soon be available in the PG

**Comment by Pryme8** [comment]
hmmm, Odd I thought I tried it in the localdev and was getting no response from it... I must have been over looking something.

**Comment by deltakosh** [comment]
yeah I end up having some issues with it recently. @sebavan  will check why soon :)

**Pull request merged**